parm/use_cases/model_applications/convection_allowing_models/GridStat_fcstFV3_obsGOES_BrightnessTempDmap.conf

```diff
@@ -0,0 +1,59 @@
+[config]
+# Loop by model initialization
+LOOP_BY = init
+
+INIT_TIME_FMT = %Y%m%d%H
+
+# Start time for METplus run
+INIT_BEG = 2019052100
+
+# End time for METplus run
+INIT_END = 2019052100
+
+# Increment between METplus runs in seconds. Must be >= 60
+INIT_INCREMENT = 3600
+
+LOOP_ORDER = processes
+
+PROCESS_LIST = GridStat(lsm1), GridStat(mp1)
+
+# Sequence of leads to process
+LEAD_SEQ = 1,2
+
+FCST_IS_PROB = false
+
+# Grid stat variables
+FCST_GRID_STAT_VAR1_NAME = SBTA1613_topofatmosphere
+FCST_GRID_STAT_VAR1_LEVELS = "(*,*)"
+FCST_GRID_STAT_VAR1_THRESH = le235
+FCST_GRID_STAT_VAR1_OPTIONS = file_type = NETCDF_MET;
+
+FCST_IS_PROB = false

```

Comment:
This line is a duplicate of line 23. Please remove one of them.

Reply:
Will do